clearwater_config_access/rph_json_config_plugin.py

```diff
@@ -0,0 +1,63 @@
+# Copyright (C) Metaswitch Networks 2017
```

Comment:
Do these files need # @file at the top?
(Others in the directory (such as [this](https://github.com/Metaswitch/clearwater-etcd-plugins/blob/master/clearwater_config_access/enum_json_config_plugin.py)) didn't so I didn't add this - let me know if they need them and I'll add them to all the files.)

Reply:
I've had a look at various python files. None of them do it so it's fine to leave it, I think.